packages/react-components/react-charts-preview/stories/VerticalBarChart/VerticalBarChartDefault.stories.tsx

```diff
@@ -120,7 +126,7 @@ export const VCBasic = () => {
       x: 80000,
       y: 20000,
       legend: 'Elephants',
-      color: DefaultPalette.blue,
+      color: 'dodgerblue',
```

Comment:
use unique shades of blue for each legend

Reply:
Previously also, they have used same shade for more than 1 legend, so followed that

Reply:
that was a bug. Could you fix it.

Reply:
Okay

Reply:
Made the change